tests/integration/tests.mk

```diff
@@ -32,6 +32,13 @@ ifneq ($(TAG),)
     _INTEGRATION_TEST_FLAGS += --istio.test.tag=$(TAG)
 endif
 
+ifeq ($(TEST_SELECT),)
+    # Default exclusions.
+    _INTEGRATION_TEST_FLAGS += --istio.test.select=-postsubmit,-flaky,-multicluster
```

Comment:
won't this disable all of our postsubmit tests that we have now? since we are not overriding for postsubmit

Reply:
I didn't see any targets that have postsubmit ... are they in a different `.mk`?

Reply:
the `test.integration.%.kube` are used though: https://github.com/istio/test-infra/blob/master/prow/config/jobs/istio.yaml#L91. They don;'t explicitly list postsubmit since it is default

Reply:
ah ok ... will sort something out.